lib/Constants.js

```diff
@@ -416,6 +416,41 @@ ReportBlock.init({
   },
 });
 
+/**
+ * Possible data types for reports.  Each report type requires data in a specific representation.
+ * For instance, collision reports require collision data, while study-related reports require
+ * data from specific types of studies.
+ *
+ * This enum also captures compatibility relations, which can be non-symmetric.  For instance,
+ * speed / volume ATR data can be used as though it were volume ATR data, but the reverse is
+ * not true.
+ */
+class ReportDataType extends Enum {
```

Comment:
this _might_ make `reportExportMode` on `StudyType` obsolete - but we should make that change separately; this PR is already plenty big enough!